addons/hr/views/hr_employee_views.xml

```diff
@@ -335,13 +342,16 @@
                                         <i class="fa fa-fw me-2 fa-phone text-primary" title="Phone"/>
                                         <field name="work_phone" />
                                     </li>
+                                    <field name="category_ids" widget="many2many_tags" options="{'color_field': 'color'}" optional="hide"/>
                                 </ul>
                             </div>
                             <div class="oe_kanban_content o_hr_employee_kanban_bottom position-absolute bottom-0 start-0 end-0">
-                                <div class="o_kanban_record_bottom">
+                                <div class="o_kanban_record_bottom mt-3">
                                     <div class="oe_kanban_bottom_left"/>
-                                    <div class="oe_kanban_bottom_right float-end pe-auto">
-                                        <widget name="hr_employee_chat"/>
+                                    <div class="oe_kanban_bottom_right">
+                                        <div class="hr_avatar mb-1 ms-2 me-n1">
+                                            <field name="user_id" widget="many2one_avatar_user" readonly="1"/>
+                                        </div>
```

Comment:
I'm wondering if we are supposed to replace it or have both :thinking: IMO it's clearer to have the discuss icon, though it'll be up to our PO to decide

Reply:
The spec said - change